pkgs/servers/zoneminder/default.nix

```diff
@@ -115,7 +115,8 @@ in stdenv.mkDerivation rec {
 
     for f in misc/*.policy.in \
              scripts/*.pl* \
-             scripts/ZoneMinder/lib/ZoneMinder/Memory.pm.in ; do
+             scripts/ZoneMinder/lib/ZoneMinder/Memory.pm.in \
+             scripts/ZoneMinder/lib/ZoneMinder/Event.pm; do
```

Comment:
```
substituteStream(): WARNING: pattern '/usr/bin/perl' doesn't match anything in file 'scripts/ZoneMinder/lib/ZoneMinder/Event.pm'
substituteStream(): WARNING: pattern '/bin:/usr/bin' doesn't match anything in file 'scripts/ZoneMinder/lib/ZoneMinder/Event.pm'
```

Reply:
Left as is due to criticality of specified patterns and high chance of them showing again given past experience.

Reply:
With this PR you propose to do two substitutions in scripts/ZoneMinder/lib/ZoneMinder/Event.pm and neither of them was needed at the time. If we were to merge this right now, this would be removed by the next attentive person unless you provide a reference for why this is needed (e.g. "we expect it to break in the future because ...").

Reply:
When I initially packaged zoneminder, there were tons of replacement warnings as it was easier to simply loop through all the files that might need fixing and fix up all the paths that had to be replaced somewhere because there soooo many and having to manually specify it for each file was a royal pain. The ZM code base is just a mess.

I suggest sticking a big fat disclaimer in there about that and simply accept the warnings.

Then again, I'm also hoping that somebody will maintain ZM as I'm not using it any more.

Reply:
Unfortunately this was a simple bit of paid work - it was hacked up locally by overwriting files in nix store, but provided back as a fix because we (me and the client) both care about upstreaming fixes.

I'd honestly prefer @peterhoeg solution of big fat warning coupled with looping over all Perl files in ZM, as the random calls to hardcoded absolute paths are a bit of a problem.

Reply:
Please amend the PR to include the note - this will be good to go then.